src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -283,16 +262,20 @@ def test_query_model(device):
 @pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU", reason="Device independent test")
 def test_register_plugin():
     core = Core()
+    if "Intel" not in core.get_property("CPU", "FULL_DEVICE_NAME"):
+        pytest.skip("This test runs only on openvino intel cpu plugin")
     core.register_plugin("openvino_intel_cpu_plugin", "BLA")
     model = core.read_model(model=test_net_xml, weights=test_net_bin)
-    exec_net = core.compile_model(model, "BLA")
-    assert isinstance(exec_net, CompiledModel), "Cannot load the network to the registered plugin with name 'BLA'"
+    compiled_model = core.compile_model(model, "BLA")
+    assert isinstance(compiled_model, CompiledModel), "Cannot load the network to the registered plugin with name 'BLA'"
 
 
 @pytest.mark.dynamic_library()
 @pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU", reason="Device independent test")
 def test_register_plugins():
     core = Core()
+    if "Intel" not in core.get_property("CPU", "FULL_DEVICE_NAME"):
+        pytest.skip("This test runs only on openvino intel cpu plugin")
```

Comment:
can we have 2 different files for macOS which we will select depending on target platform? For arm64, we can use file with arm plugin, for x86 - current file.

Reply:
added